### DISCUSSION TITLE
What's your opinion on this?

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [TechStudent10/techstudent10#17] [General]
Hey, 

I'm just seeing how the gradient background colors look on the buttons in Mogul Christmas. I kinda like it, but I wanted your opinion on it. Do you like the new colors or do you like to solid colors for the buttons?

Solid colors: https://mogulchristmas.kendalldoescoding.gq/
Gradient: https://deploy-preview-152--mogulchristmas.netlify.app/ or https://mogulchristmas.kendalldoescoding.gq/gradientbg

Please let me know and I'll merge/close PR https://github.com/KendallDoesCoding/mogul-christmas/pull/152 accordingly

**Comment by TechStudent10** [comment]
Those look awesome. Just maybe make the gradients a bit more like the solid colors (i.e the top button has a red gradient). Then you'll be all good.

**Reply by KendallDoesCoding** [reply]
Ok thanks a lot, I'll play around with the gradients and ask you again if its good to merge.

**Reply by KendallDoesCoding** [reply]
You too can play around with the colors and tell me if you think that color will look good, the website is https://cssgradient.io/ I rn used their swatches but you can make your own gradients using the main url. So, I'll work on that shortly.

**Comment by KendallDoesCoding** [comment]
I made, https://deploy-preview-154--mogulchristmas.netlify.app/ as a alternative to this, I need to update the first button color though.

**Reply by KendallDoesCoding** [reply]
Yup, updated the first button now, let me know if you like the new colors or no.

**Reply by TechStudent10** [reply]
The top one isn't updated for me.

**Reply by KendallDoesCoding** [reply]
No, I made it light. I am gonna work on another design for these gradients.

**Reply by TechStudent10** [reply]
ohhh okay

**Comment by KendallDoesCoding** [comment]
I made this a little more like the original but a little small gradient color. Do you like it? https://deploy-preview-157--mogulchristmas.netlify.app

**Reply by KendallDoesCoding** [reply]
I like the ludwig socials button, will change the above 2 though

**Comment by KendallDoesCoding** [comment]
What do you think of this? https://github.com/KendallDoesCoding/mogul-christmas/discussions/168

**Comment by KendallDoesCoding** [comment]
Finally got some colours I like and commited.  What's your opinion on it? Let me know here: https://github.com/KendallDoesCoding/mogul-christmas/discussions/204

**Comment by KendallDoesCoding** [comment]
Do you like the badges in the README.md?

https://github.com/KendallDoesCoding/mogul-christmas

**Reply by TechStudent10** [reply]
that looks nice

**Reply by KendallDoesCoding** [reply]
thanks!

**Comment by KendallDoesCoding** [comment]
Please review how the project looks after this issue, info in issue with commit relating to change, etc. https://github.com/KendallDoesCoding/mogul-christmas/issues/211

**Comment by KendallDoesCoding** [comment]
Do you like this?

![image](https://user-images.githubusercontent.com/84712013/157071860-4f98155a-51e4-41b8-b01f-c7895e43aba8.png)

**Reply by TechStudent10** [reply]
that looks good

**Comment by KendallDoesCoding** [comment]
Please read your YT email.

**Reply by TechStudent10** [reply]
what???? how did you get hold of my email?

**Reply by KendallDoesCoding** [reply]
you used the chat plugin on my site and filled it in, assumed it was a public email? if your not comfertable will share it here it was easier to share in email as i just had to forward something.

**Reply by KendallDoesCoding** [reply]
Okay, I'll just ask the question here and not email ya in future.

What backgrounds do you like better?

![Photo 27-3-22, 2 49 19 am](https://user-images.githubusercontent.com/84712013/160289823-92b828d6-3cfc-4932-b320-3dc6d4ba6d74.png)
![Photo 27-3-22, 2 49 30 am](https://user-images.githubusercontent.com/84712013/160289826-99bba13e-87c1-44a4-8e8d-bf52c3455684.png)
![Photo 27-3-22, 2 49 57 am](https://user-images.githubusercontent.com/84712013/160289827-406972ef-471e-4f80-904f-21f99dbed8bf.png)
![Photo 27-3-22, 2 49 47 am](https://user-images.githubusercontent.com/84712013/160289829-37f8d8a7-b401-4641-af0e-2e25d15c9874.png)

**Reply by TechStudent10** [reply]
2nd and 3rd for sure.

**Reply by KendallDoesCoding** [reply]
Ok, currently for lightmode we use 4 and for darkmode we use 3, but get what you mean, will play around with the colour of the background tommrow.

**Comment by KendallDoesCoding** [comment]
Please review: https://github.com/KendallDoesCoding/mogul-christmas/pull/284
